.github/workflows/pelican-action-test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   pelican-test:
     env: # source and target branches
       SOURCE: testsite
-      TARGET: testsite-${{ github.ref_name }} # each branch has own test output
+      TARGET: testsite-${{ github.ref_name }} # each branch has its own test output
```

Comment:
This is a grammar error that I tripped on while looking at this [workflow failure](
https://github.com/jsoref/infrastructure-actions/actions/runs/13163485494/workflow)